libs/dh/imbalance-prices/shell/src/lib/table-day-view/dh-table-day-view.component.ts

```diff
@@ -53,13 +57,17 @@ import { DhImbalancePricesForDay, DhImbalancePricesForDayProcessed } from '../dh
       </ng-container>
 
       <ng-container *wattTableCell="columns['price']; let entry">
-        <span [ngClass]="{ 'negative-price': entry.price < 0 }">
-          {{ entry.price | number: '1.5-6' }}
-        </span>
+        @if (entry.price === 0) {
```

Comment:
question: can there ever be a price that is actually 0? In that case the UI will show a missing price badge. Wouldn't it be better if the prices was `null`/`undefined`?

Reply:
The DTO does not allow for null / undefined. When we need to change the data structure in markpart @FirestarJes What do you think?

Reply:
I don't think there can actually ever be a "0" price, that would be missing a price, but should probably consult Irene on this one, as she will be able to answer this.

Reply:
If the price can never be exactly 0, then it's fine to keep it like that.

Reply:
Prices can in rare cases be 0, so need to find another solution

Reply:
Is now set as null